pkg/builtinchecks/yamls/pdbs-min-available.yaml

```
@@ -0,0 +1,7 @@
+name: "pdb-min-available"
+description: "Indicates when a DeploymentLike object has a \"Pod Disruption Budget\" specified with a MinAvailable value that is too restrictive."
+remediation: "Change the \"Pod Disruption Budget\" to have a \"MinAvailableValue\" set to number lower than the Replica count in the related DeploymentLike. Refer to https://kubernetes.io/docs/tasks/run-application/configure-pdb/ for more information."
```

Similar as for the other check.

Option 1:

```suggestion
description: "Indicates when a deployment-like object has a PodDisruptionBudget specified with a minAvailable value that prevents disruptions."
remediation: "Change the PodDisruptionBudget to have minAvailable set to a number lower than the number of replicas in the related deployment-like objects. Refer to https://kubernetes.io/docs/tasks/run-application/configure-pdb/ for more information."
```

Option 2 (turning the description around):

```suggestion
description: "Indicates when a PodDisruptionBudget sets a minAvailable value that will always prevent disruptions of pods created by related deployment-like objects."
remediation: "Change the PodDisruptionBudget to have minAvailable set to a number lower than the number of replicas in the related deployment-like objects. Refer to https://kubernetes.io/docs/tasks/run-application/configure-pdb/ for more information."
```